conf/denas/asr/e2eaiok_denas_asr_pruner_train.conf

```diff
@@ -0,0 +1,105 @@
+seed: "74443"
+output_folder: "results/transformer/74443"
+save_folder: "results/transformer/74443/save"
+device: cpu
+dist_backend: gloo
+mode: train
+domain: asr
+
+# Data files
+data_folder: "/home/vmagent/app/dataset/LibriSpeech"
+skip_prep: false
+train_csv: "/home/vmagent/app/dataset/LibriSpeech/train.csv"
+valid_csv: "/home/vmagent/app/dataset/LibriSpeech/dev-clean.csv"
+test_csv: "/home/vmagent/app/dataset/LibriSpeech/test-clean.csv"
+tokenizer_ckpt: "/home/vmagent/app/dataset/LibriSpeech/tokenizer.ckpt"
+pre_trained: true
+model: "/home/vmagent/app/dataset/LibriSpeech/asr.ckpt"
+pruner:
+    pruner: true
+    backend: pytorch
+    algo: l1unstructured
+    layer_list:
+    exclude_list:
+    sparsity: /home/vmagent/app/e2eaiok/e2eAIOK/DeNas/best_model_structure.txt
+
+ckpt_interval_minutes: 30 # save checkpoint every N min
+
+# Training parameters
+train_epochs: 60
+eval_epochs: 1
+train_batch_size: 32
+eval_batch_size: 1
+num_workers: 1
+ctc_weight: 0.3
+grad_accumulation_factor: 1
+max_grad_norm: 5.0
+loss_reduction: 'batchmean'
+sorting: random
+metric_threshold: 5.8
+
+lr_adam: 0.001
+
+# Feature parameters
+sample_rate: &sample_rate 16000
+n_fft: &n_fft 400
+n_mels: &n_mels 80
+
+####################### Model parameters ###########################
```

Comment:
What's this CNN?

Reply:
CNN here is a sub module of ASR to process audio features